recipes/te-aid/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "TE-Aid" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "2150a8bb7a16ceba29f824063dd7866da9e827e952b1dbf01fb61f0c89b18904" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bricoletc/TE-Aid/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  run:
+    - blast >=2.15.0
+    - emboss >=6.6.0
+    - r >=4.2
+
```

Comment:
From https://github.com/clemgoub/TE-Aid/blob/main/TE_AID.yml

  - r-rcpp >=1.0.12
  - bioconductor-biostrings >=2.70.1
  
  Not sure if there is a diff between r and r-base from conda-forge